Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerSuggestionsViewController_Tests.swift

```diff
@@ -0,0 +1,246 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChatTestTools
+@testable import StreamChat
+@testable import StreamChatUI
+import XCTest
+
+class ChatMessageComposerSuggestionsViewController_Tests: XCTestCase {
+    
+    // We need to provide a size to the suggestions view since here we are testing the view in isolation,
+    // and so we can't attach it to a bottomAnchorView. The test to verify the height calculation dependent
+    // on the rows should be done in the parent view controller tests.
+    private let defaultSuggestionsSize = CGSize(width: 360, height: 130)
+    
+    // MARK: - Mock Data
+    
+    private let commands: [Command] = [
+        .init(name: "giphy", description: "", set: "", args: "[text]"),
+        .init(name: "default", description: "", set: "", args: "[@username] [text]")
+    ]
+    
+    private let mentions: [ _ChatUser<NoExtraData>] = [
+        .mock(
+            id: "vader",
+            name: "Mr Vader",
+            imageURL: TestImages.vader.url,
+            isOnline: false
+        ),
+        .mock(
+            id: "yoda",
+            name: "Yoda",
+            imageURL: TestImages.yoda.url,
+            isOnline: true
+        )
+    ]
+    
+    var vc: ChatMessageComposerSuggestionsViewController!
+    
+    override func setUp() {
+        super.setUp()
+        vc = ChatMessageComposerSuggestionsViewController()
+        vc.uiConfig.images.commandIcons["giphy"] = TestImages.commandGiphy.image
+        vc.uiConfig.images.messageComposerCommandFallback = TestImages.commandFallback.image
```

Comment:
You can't do this. This modifies the default `UIConfig`. You need to create a new `UIConfig` instance and assign it explicitly to the VC. We do it already in all other tests like that.